microsoft/PowerToys
# [FancyZones] Canvas layouts scaling

## Summary of the Pull Request

**What is this about:**

Enable canvas layouts scaling. 
This allow us to remove resolution from `device-id` which causes many issues with layout resetting. 

**What is included in the PR:** 

**How does someone test / validate:** 

* Create and apply canvas layout.
* Apply the same canvas layout on the monitor with different resolution, verify it looks the same.

## Quality Checklist

- [x] **Linked issue:** #14261

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
